amplipi/auth.py

```diff
@@ -2,7 +2,7 @@
 import json
 import time
 import secrets
-import logging
+from app import logger
```

Comment:
you probably actually wanna use `getLogger`: https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial